scripts/release/debian/test_deb_package.py

```diff
@@ -7,7 +7,8 @@
 import sys
 import subprocess
 
-root_dir = '/opt/az/lib/python3.10/site-packages/azure/cli/command_modules'
+python_minor_version = sys.version_info[1]
+root_dir = f'/opt/az/lib/python3.{python_minor_version}/site-packages/azure/cli/command_modules'
```

Comment:
This is far more elegant than the previous code. Why isn't the failure noticed in https://github.com/Azure/azure-cli/pull/26749?